sr_robot_commander/test/test_robot_commander.py

```diff
@@ -402,281 +427,182 @@ def test_make_named_trajectory(self):
 
     def test_send_stop_trajectory_unsafe(self):
         self.reset_to_home()
```

Comment:
Given that you have deleted the previous ones, just making sure you really want to keep this one? Especially that I don't think we need to be in home pose to carry out this test right?